source/agora/utils/TxBuilder.d

```diff
@@ -228,9 +231,46 @@ public struct TxBuilder
         return this;
     }
 
+    /***************************************************************************
+
+        Sets the unlocker function to sign the inputs
+
+        If not set then the default unlocker using WellKnownKeys is used.
+
+        Params:
+            unlocker = function to sign the inputs of the transaction
+
+        Returns:
+            A reference to `this` to allow for chaining
+
+    ***************************************************************************/
+
+    public ref typeof(this) unlockSigner (Unlock
+        function (in Transaction tx, in OutputRef out_ref) @safe nothrow unlocker) return scope
+        @safe nothrow @nogc pure
+    {
+        this.unlocker = unlocker;
+        return this;
+    }
+
+    public static Unlock signWithGenesis (in Transaction tx, in OutputRef out_ref)
+        @safe nothrow
+    {
+        auto pair = SigPair(WK.Keys.Genesis.sign(tx.getChallenge()), SigHash.All);
+        return Unlock([ubyte(65)] ~ pair[].dup);
+    }
+
+    // Sign with a given key and append the given OpCode to the unlock bytes
```

Comment:
Two slashes are comment, 3 slashes are DDOC

Reply:
Removed from this PR.